public/sw.js

```diff
@@ -23,15 +23,27 @@ const fetchEventHandler = async event => {
     if (req.headers.has('accept')) {
       supportsWebp = req.headers.get('accept').includes('webp');
     }
+    // detect the ios version and remove webp from the request
```

Comment:
This comment is probably misleading as we don't actually detect he IOS version we just rely on the accept header.

Reply:
will remove